ui/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/ProcessVersionEntityFactory.scala

```diff
@@ -47,12 +45,6 @@ trait ProcessVersionEntityFactory extends BaseEntityFactory {
     def modelVersion: Rep[Option[Int]] = column[Option[Int]]("model_version", NotNull)
 
     def pk = primaryKey("pk_process_version", (processId, id))
-
-    private def process: ForeignKeyQuery[ProcessEntityFactory#ProcessEntity, ProcessEntityData] = foreignKey("process-version-process-fk", processId, processesTable)(
```

Comment:
this foreign key isn't  used in some DDLs created base on class definition?

Reply:
Right.. I turn it back.